packages/react-ui-components/src/Tabs/tabs.tsx

```diff
@@ -120,6 +121,7 @@ export default class Tabs extends PureComponent<TabsProps> {
 
                     return (
                         <div
+                            className={theme!.tabs__panel}
```

Comment:
```suggestion
                            className={theme.tabs__panel}
```
since this `tabs_panel` is already in TabsTheme, `!` shouldn't be necessary here, right?

Reply:
In this case I can't build the package:
```
ERROR in /Packages/Application/Neos.Neos.Ui/packages/react-ui-components/src/Tabs/tabs.tsx(124,40)
TS2532: Object is possibly 'undefined'.
```

Reply:
Wait a minute, but it IS possibly undefined! The theme object is marked as optional here: https://github.com/neos/neos-ui/blob/973d575154b48c8b0f5593abd0010abd656aafdc/packages/react-ui-components/src/Tabs/tabs.tsx#L27

But since we export the Tabs component wrapped in `themr`, the `theme` prop would always be there: https://github.com/neos/neos-ui/blob/973d575154b48c8b0f5593abd0010abd656aafdc/packages/react-ui-components/src/Tabs/index.ts#L6

So I'd say just mark the `theme` object as required.

Reply:
Done. Thanks for the help :)